docs/ui/rate-limits.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Prefect Cloud API rate limits.
+tags:
+    - API
+    - Prefect Cloud
+    - rate limits
+---
+
+# Prefect Cloud API Rate Limits
+
+An API rate limit is a way for Prefect to ensure the stability of the Prefect Cloud platform. API rate limits enable us to control the number of requests any given app can make on the platform. It ensures that, when you make an API call, you get a response.
```

Comment:
```suggestion
API rate limits restrict the number of requests that a single client can make in a given time period. They ensure Prefect Cloud's stability, so that when you make an API call, you always get a response.
```